src/ElectronApp.ts

```diff
@@ -30,7 +30,7 @@ export default class ElectronApp {
     this.autoUpdater = new AutoUpdater()
     this.protocol = process.env.NODE_ENV === 'development' ? DEEP_LINK_PROTOCOL_DEV : DEEP_LINK_PROTOCOL
 
-    if (!this.app.requestSingleInstanceLock()) {
```

Comment:
this one is broken the build

Reply:
Hmm, I never added this, its maybe from your side?

Reply:
Ohh i see sorry